node/commons/compatibility-layer.ts

```diff
@@ -279,6 +287,16 @@ const fillSearchItemWithSimulation = (searchItem: SearchItem, orderFormItems: Or
         PaymentSystemGroupName: '',
       }]
     })
+
+    const sellerDefault = getSellerDefault(searchItem.sellers)
+    if (sellerDefault && searchItem.sellers.find(seller => seller.sellerDefault)?.sellerId !== sellerDefault) {
```

Comment:
do we need this "find"? I think it's the same complexity of just returning the map

Reply:
I just wanted to ensure that we don't change anything if the default seller is already correct.. what do you think?